src/main/java/reposense/parser/CsvParser.java

```diff
@@ -77,6 +80,9 @@ public CsvParser(Path csvFilePath) throws IOException {
         return results;
     }
 
+    /**
+     * Checks if there is malformed line.
```

Comment:
`Checks if the {@code line} contains values at the mandatory positions in CSV format.`